homeassistant/components/local_todo/todo.py

```diff
@@ -63,9 +63,11 @@ def _todo_dict_factory(obj: Iterable[tuple[str, Any]]) -> dict[str, str]:
     """Convert TodoItem dataclass items to dictionary of attributes for ical consumption."""
     result: dict[str, str] = {}
     for name, value in obj:
+        if value is None:
```

Comment:
Will pull this into a separate PR https://github.com/home-assistant/core/pull/104182